Marlin/Marlin_main.cpp

```diff
@@ -5038,7 +5038,7 @@ inline void gcode_M121() { endstops.enable_globally(false); }
    */
   inline void gcode_M156() {
     uint8_t addr = code_seen('A') ? code_value_short() : 0;
-    int bytes    = code_seen('B') ? code_value_short() : 0;
+    int bytes    = code_seen('B') ? code_value_long() : 0;
 
     if (addr && bytes) {
       i2c.address(addr);
```

Comment:
I do not know about modifying sending data over i2c to a long... It would be converted BACK to
an uint8 from 32 bits during the call by the compiler... But I would remove that change.

If it was part of my 1311 then I did not need to change that to get it to work for me.